lib/TPP/Dialect/Xsmm/XsmmUtils.cpp

```diff
@@ -8,18 +8,67 @@
 
 #include "TPP/Dialect/Xsmm/XsmmUtils.h"
 #include "TPP/Dialect/Xsmm/XsmmOps.h"
+#include "TPP/Transforms/Utils/VNNIUtils.h"
 #include "TPP/Transforms/Utils/ValueUtils.h"
 #include "mlir/Dialect/Func/IR/FuncOps.h"
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
+#include "mlir/Dialect/MemRef/IR/MemRef.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
+#include "mlir/Dialect/Utils/IndexingUtils.h"
+#include "mlir/Dialect/Vector/IR/VectorOps.h"
 #include "mlir/IR/BuiltinTypeInterfaces.h"
 #include "mlir/IR/PatternMatch.h"
 #include "mlir/IR/TypeUtilities.h"
 
 #include "llvm/ADT/STLExtras.h"
+#include "llvm/ADT/SetOperations.h"
 #include "llvm/Support/Compiler.h"
+#include "llvm/Support/Debug.h"
+#include <map>
+#define DEBUG_TYPE "xsmm-utils"
+
+using namespace mlir;
```

Comment:
this is unnecessary, as below you're already in the `mlir` namespace.

Please don't use `using namespace` in these files, as it complicates namespace resolution. Use the existing pattern of entering and leaving namespaces as needed.

Reply:
Removed the namespace mlir in XsmmUtils.cpp. However, I haven't removed it in other files because that meant namespace resolution for a bunch of objects there.